NetKVM/ProtocolService/ProtocolService.cpp

```diff
@@ -739,6 +739,10 @@ class CProtocolServiceImplementation :
     {
         StopThread();
         m_ThreadEvent.Set();
+        while (IsThreadRunning())
```

Comment:
Check if this prolongs shutdown\reboot of the system.

Reply:
Does not, checked